Rubberduck.Core/UI/Controls/SearchResultWindow.cs

```diff
@@ -21,10 +22,8 @@ public ISearchResultsWindowViewModel ViewModel
             }
         }
 
-        public string ClassId
-        {
-            get { return "BFD04A86-CACA-4F95-9656-A0BF7D3AE254"; }
-        }
+        private readonly string RandomGuid = Guid.NewGuid().ToString();
```

Comment:
If we *do* need a GUID, then why can't it be a `RubberduckGuid` constant?

Reply:
This particular GUID is just used as an unique identifier for an instance created by the `CreateToolWindow`, as [per documentation](https://msdn.microsoft.com/en-us/vba/language-reference-vba/articles/createtoolwindow-method) for `GuidPosition` parameter.

Reply:
That said, `RandomGuid` really does not convey the reason. I would have preferred a name like `InstanceIdentifier` or something similar.